plasma_framework/test/src/exits/payment/PaymentProcessInFlightExit.test.js

```diff
@@ -354,14 +339,18 @@ contract('PaymentInFlightExitRouter', ([_, ifeBondOwner, inputOwner1, inputOwner
 
                 this.dummyExitId = 666;
                 this.exit = await buildInFlightExitData();
+                await this.exitGame.setInFlightExit(this.dummyExitId, this.exit);
```

Comment:
Everywhere in this file we use dummyExitId with value 666. How about, we put it into a constant?